### PR TITLE
Update default-rules.yaml

### PR DESCRIPTION
**Pull request by INT2ECALL**
fix bypass by serviceaccount group

### CODE REVIEW

File: pkg/analysis/default-rules.yaml [hunk 1]
```diff
@@ -52,7 +52,7 @@ Rules:
         Comment: "Exclude kube-system from analysis"
         Disabled: true
         Expression: |
-          has(subject.namespace) && (subject.namespace == "kube-system")
+          has(subject.name) && subject.name.startsWith('system:') && (subject.name != "system:serviceaccounts")
```
Comment (gadinaor-r7) [inline pkg/analysis/default-rules.yaml:55]: The exclusion rule intent is avoid reporting on objects in kube-system -this change completely change this intent

**Review by gadinaor-r7 — CHANGES_REQUESTED** [review]
See comments

File: pkg/analysis/default-rules.yaml (hunk 1 above)
Comment (INT2ECALL) [reply pkg/analysis/default-rules.yaml:55]: yes,i know ,but all serviceaccount  can give some privilege ,i found some customer use this

Comment (gadinaor-r7) [reply pkg/analysis/default-rules.yaml:55]: @INT2ECALL in that case you can create a private analysis rules, tailored to your customer with the relevant changes and pass to the analysis run.

`rbac-tool analyze --config your-rules.yaml`

Comment (INT2ECALL) [reply pkg/analysis/default-rules.yaml:55]: @gadinaor-r7 cool ，thanks

**Pull request closed without merging**